sips/sip-022/sip-022-emergency-pox-fix.md

```diff
@@ -0,0 +1,279 @@
+# Preamble
+
+SIP Number: 022
+
+Title: Emergency Fix to PoX Stacking Increases
+
+Authors:
+    Aaron Blankstein <aaron@hiro.so>,
+    Jude Nelson <jude@stacks.org>
+
+Consideration: Technical, Governance
+
+Type: Consensus
+
+Status: Draft
+
+Created: 19 April 2023
+
+License: BSD 2-Clause
+
+Sign-off: 
+
+Discussions-To: https://github.com/stacksgov/sips
+
+# Abstract
+
+On 19 April 2023, it was discovered that there was a bug in the PoX smart
+contract which would allow a user to claim that they have stacked far, far more
+STX than they had locked.  Exploiting this bug both allows the user to increase
+their PoX reward slot allotment, while also driving up the stacking minimum.
+Furthermore, it creates the conditions for a network-wide crash:  if the total
+amount of STX stacked as reported by the PoX smart contract were to ever exceed
+the total amount of liquid STX, then the node would crash into an irrecoverable
+state.  This bug has already been triggered in the wild.
+
+This SIP proposes **two immediate consensus-breaking changes** that both prevents this bug
```

Comment:
both prevent

Reply:
this is grammatically correct, no change is needed here